src/store/store-manager.js

```diff
@@ -0,0 +1,41 @@
+import store from ".";
+import ApiClient from "../services/api-client.js";
+
+export default {
+  // poller test
+  poll() {
+    console.log("polledd...");
+    console.log(JSON.stringify(store.getters.feed));
+    // setTimeout(() => this.poll(), 1000)
+    return;
+  },
+
+  // checkItem() queries item and updates itemList, returning list of resultIds
+  checkItem(itemId) {
+    let item = store.getters.trackedItems[itemId];
```

Comment:
Oh wait, so ur gonna keep track of stored items?

Reply:
this is you